File: 8 kyu/double_char.js
// link : https://www.codewars.com/kata/56b1f01c247c01db92000076

function doubleChar(str) {
    let array = str.split("");
    let newArray = [];
    for (let i=0; i<array.length; i++) {
      newArray.push(array[i]);
      newArray.push(array[i]);
    }
    return newArray.join("");
  }
  

File: 8 kyu/DoubleChar.java
// link : https://www.codewars.com/kata/56b1f01c247c01db92000076/train/java

public class Solution{
  public static String doubleChar(String s){
    String doubleString = "";
    for (int i=0; i<s.length(); i++) {
      doubleString += s.charAt(i);
      doubleString += s.charAt(i);
    }
    return doubleString;
  }
}
